.github/ISSUE_TEMPLATE/moderator_application.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Community Moderator Request
```

Comment:
Can we pull this file out into a separate PR? I don't want to hold up the other content, but we don't have issue templates on k/community, and if we add our first one, it will change the workflow for opening new issues completely.